cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultSchedulerService.java

```diff
@@ -60,24 +69,42 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
       String[] parts = key.split(":");
       Preconditions.checkArgument(parts.length == 5);
 
-      String accountId = parts[0];
+      String namespaceId = parts[0];
       String applicationId = parts[1];
       ProgramType programType = ProgramType.valueOf(parts[2]);
       String programId = parts[3];
       String scheduleName = parts[4];
 
       LOG.debug("Schedule execute {}", key);
-      Arguments args = new BasicArguments(ImmutableMap.of(
-        ProgramOptionConstants.LOGICAL_START_TIME, Long.toString(context.getScheduledFireTime().getTime()),
-        ProgramOptionConstants.RETRY_COUNT, Integer.toString(context.getRefireCount()),
-        ProgramOptionConstants.SCHEDULE_NAME, scheduleName
-      ));
+      ImmutableMap.Builder<String, String> builder = ImmutableMap.builder();
+
+      builder.put(ProgramOptionConstants.LOGICAL_START_TIME, Long.toString(context.getScheduledFireTime().getTime()));
+      builder.put(ProgramOptionConstants.RETRY_COUNT, Integer.toString(context.getRefireCount()));
+      builder.put(ProgramOptionConstants.SCHEDULE_NAME, scheduleName);
+      String schedulerQueue = getNamespaceResolvedScheduleQueue(namespaceId);
+      if (schedulerQueue != null) {
+        builder.put(Constants.AppFabric.APP_SCHEDULER_QUEUE, schedulerQueue);
```

Comment:
Yes. I noticed that the triggered jobs we submitted to default queues. that's when I found this missing piece.